src/app/meetups/meetups-add.component.html

```diff
@@ -0,0 +1,14 @@
+<form class="form-horizontal" (ngSubmit)="onSubmit(meetupForm.value)" #meetupForm="ngForm">
+    <legend>Start a New Meetup</legend>
```

Comment:
please fix indentation, and do use reactive form and validation

Reply:
Could you please add translation ?

Reply:
still working on meetup form ( work in progress )  , added to TODO list. Thanks for feedback